alerta/app/views.py

```diff
@@ -723,21 +723,35 @@ def create_user():
 @admin_required
 @jsonp
 def update_user(user):
-
+   
+    name, login, password, provider, text, email_verified = None, None, None, None, None, None
+       
```

Comment:
This can be replaced with something like ...
```
provider = request.json.get("provider", None)
```
... similar to how it's done in the `create_blackout()` method.